DemoArrayList/src/DemoList/OrangeArrays.java

```diff
@@ -0,0 +1,41 @@
+package DemoList;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Kanini W'Tang on 7/18/2017.
+ */
+public class OrangeArrays {
+
+    public void ArrayListDemo (){
```

Comment:
A method name must be an action word(s). and a method must start with lower case.